src/routes/recipes/+page.svelte

```diff
@@ -152,6 +152,21 @@
 		top: 4px;
 	}
 
+	.list-title::before {
```

Comment:
It's a matter of preferences, but I would use `after` instead of `before`, because the line you add is **after** the content